spec/serializers/rest/admin/measure_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe REST::Admin::MeasureSerializer, pending: 'broken' do
```

Comment:
For this one - the `retrieve`  method of `Measure` is going to return an array of measures, of varying class types. Its fine to leave the setup that way and assert about the first  instance or something, but you could also just used one of the classes, ie:

`let(:record) { Admin::Metrics::Measure::ActiveUsersMeasure.new(start_at, end_at, nil) }`

Have not double checked but I think they are set up that way.

Then the assertions might also need updating re: whatever the returned json format is here.

Reply:
Thanks. Updated and fixed.